[2024-09-25T10:00:16.971+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T10:00:17.004+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.warehouse.step_1.load.categories scheduled__2024-09-24T00:00:00+00:00 [queued]>
[2024-09-25T10:00:17.024+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.warehouse.step_1.load.categories scheduled__2024-09-24T00:00:00+00:00 [queued]>
[2024-09-25T10:00:17.024+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-25T10:00:17.046+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): warehouse.step_1.load.categories> on 2024-09-24 00:00:00+00:00
[2024-09-25T10:00:17.057+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=448) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T10:00:17.060+0000] {standard_task_runner.py:72} INFO - Started process 454 to run task
[2024-09-25T10:00:17.060+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'warehouse.step_1.load.categories', 'scheduled__2024-09-24T00:00:00+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmpgzig8fme']
[2024-09-25T10:00:17.063+0000] {standard_task_runner.py:105} INFO - Job 74: Subtask warehouse.step_1.load.categories
[2024-09-25T10:00:17.141+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.warehouse.step_1.load.categories scheduled__2024-09-24T00:00:00+00:00 [running]> on host 639d778bfb8f
[2024-09-25T10:00:17.286+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='warehouse.step_1.load.categories' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-24T00:00:00+00:00'
[2024-09-25T10:00:17.289+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T10:00:17.312+0000] {base.py:84} INFO - Retrieving connection 'minio'
[2024-09-25T10:00:17.348+0000] {base.py:84} INFO - Retrieving connection 'warehouse_db'
[2024-09-25T10:00:17.457+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-25T10:00:17.477+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T10:00:17.478+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=warehouse.step_1.load.categories, run_id=scheduled__2024-09-24T00:00:00+00:00, execution_date=20240924T000000, start_date=20240925T100017, end_date=20240925T100017
[2024-09-25T10:00:17.518+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-25T10:00:17.581+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-25T10:00:17.585+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
